Lib/fontTools/ufoLib/__init__.py

```diff
@@ -459,7 +459,12 @@ def readMetaInfo(self, validate=None):
 	# groups.plist
 
 	def _readGroups(self):
-		return self._getPlist(GROUPS_FILENAME, {})
+		groups = self._getPlist(GROUPS_FILENAME, {})
+		# remove any duplicate glyphs in a kerning group
+		for groupName, glyphList in groups.items():
+			if groupName.startswith('public.kern1.') or groupName.startswith('public.kern2.'):
```

Comment:
I have recently learned that you can spell multiple startswith options with a tuple of strings:
`if groupName.startswith(('public.kern1.', 'public.kern2.')):`

Reply:
Oh nice! I'll update it to use a tuple. Thanks!